recipes/openms-meta/build.sh

```
@@ -10,9 +10,25 @@ export LIBRARY_PATH=${PREFIX}/lib
 #export DYLD_LIBRARY_PATH=${PREFIX}/lib
 
-mkdir build
+
+echo "Current directory: $(pwd)"
+echo "SRC_DIR: $SRC_DIR"
+
+# Save current directory
+CURRENT_DIR=$(pwd)
+
+# Go to source directory
+cd $SRC_DIR
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)